openmdao/docs/advanced_guide/advanced_case_recording.rst

```diff
@@ -0,0 +1,86 @@
+.. _advanced_case_recording:
+
+***************************
+Advanced Recording Example
+***************************
+
+Below we demonstrate a more advanced use case of case recording including the four different objects
+that a user can attach to. We will then show how to extract various data from the model and finally,
+relate that an XDSM for illustrative purposes.
+
+.. embed-code::
+    openmdao.recorders.tests.test_sqlite_recorder.TestFeatureSqliteRecorder.test_feature_advanced_example
+    :layout: interleave
+
+
+Below we have an XDSM to show the SellarMDA component equations and their inputs and outputs. Through
+the different recorders we can access the different parts of the model. We'll take you through an
+example of each object and relate it back to the diagram below.
+
+.. image:: images/sellar_xdsm.jpg
+    :width: 600
+
+System
+-------
+First, we'll examine the `system` recorder. Suppose we want to know what the value of `y1` is going
+into the objective function (obj_func). Using the `list_cases` method, we'll query the objective
+function by passing in the string `root.obj_comp`. You could also access the discipline equations
+by swapping out the subsystem `root.obj_comp` for `root.con_cmp1`. Next we use `get_case` to
+inspect which variables are in the dictionary. Here we find that `x, y1, y2, z` are returned.
+Since we originally sought find the value of `y1` going into the objective function, we'll loop
+through the 14 cases to find what the value is in each case.
+
+.. embed-code::
+    openmdao.recorders.tests.test_sqlite_recorder.TestFeatureSqliteRecorder.test_feature_system_recorder
+    :layout: interleave
+
+Solver
+------
+
+Similar to the `system` recorder, we can query the `solver` but in this case we will find the
+calculated value of obj_func. You can also access the values of inputs to the equation with the
+solver but in this case we'll focus on the obj_func value since we cannot get that information from
+the `system` recorder.
```

Comment:
This is not exactly right... `obj` is available in the system cases as an output of the `obj_cmp` component..  

The point of the solver cases is to see the history of the solver convergence within a Group or ImplicitComponent (maybe you want to put the solver recorder on the `cycle` Group in the example for this to make more sense)...